### PR TITLE
branch tuto git

### PR DESCRIPTION
**Pull request by OlivierBernard**

### CODE REVIEW

File: collections/_posts/2022-06-28-tuto_git.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+layout: review
+title: "tutu git"
+tags: CNN transformer classification
+author: "Olivier Bernard"
+cite:
+    authors: "Olivier Bernard"
+    title:   "Escaping the big data paradigm with compact transformers"
+    venue:   "CVPR LLID Workshop 2021"
+pdf: "https://arxiv.org/abs/2104.05704"
```
Comment (sophieCarneiro) [inline collections/_posts/2022-06-28-tuto_git.md:10]: Take care, bad link

File: collections/_posts/2022-06-28-tuto_git.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+---
+layout: review
```
Comment (MehdiShekarnabi) [inline collections/_posts/2022-06-28-tuto_git.md:2]: Very good!

**Review by sophieCarneiro — CHANGES_REQUESTED** [review]

File: collections/_posts/2022-06-28-tuto_git.md (hunk 1 above)
Comment (OlivierBernard) [reply collections/_posts/2022-06-28-tuto_git.md]: Hi sophie
Thanks for having poiting out this mistake. It is fixed now !

File: collections/_posts/2022-06-28-tuto_git.md (hunk 2 above)
Comment (OlivierBernard) [reply collections/_posts/2022-06-28-tuto_git.md:2]: thanks

**Review by sophieCarneiro — APPROVED** [review]
Nice work !

**Pull request merged**